dpsim/examples/cxx/CIM/WSCC_9bus_mult_diakoptics.cpp

```diff
@@ -113,7 +113,7 @@ int main(int argc, char *argv[]) {
   filenames = DPsim::Utils::findFiles(
       {"WSCC-09_RX_DI.xml", "WSCC-09_RX_EQ.xml", "WSCC-09_RX_SV.xml",
        "WSCC-09_RX_TP.xml"},
-      "build/_deps/cim-data-src/WSCC-09/WSCC-09_RX", "CIMPATH");
+      "../../../build/_deps/cim-data-src/WSCC-09/WSCC-09_RX", "CIMPATH");
```

Comment:
I get an error when I run the simulation because of incorrect search path here. I think the path should be kept the same as before. Please update